### PR TITLE
Fix UnitTest to work in CakePHP 3.2.x

### PR DESCRIPTION
**Pull request by greew**

### CODE REVIEW

**Comment by greew** [comment]
The failing check above is that the COVERALLS test is run on a PHP 5.4, but when updating this to run on CakePHP 3.2.x we need PHP 5.5+.
Should I "just" update the travis-ci configuration to run the test on 5.5 instead of 5.4?

**Comment by ADmad** [comment]
Need to update the coveralls build to use PHP 7 instead. You can refer to https://github.com/cakephp/cakephp/blob/master/.travis.yml

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,6 @@
 language: php
 
 php:
-  - 5.4
```
Comment (ADmad) [inline .travis.yml]: We need to test on 5.4 since Cake 3.0 supports it :)

**Comment by ADmad** [comment]
Jeez don't just copy paste the travis config from core repo. Just refer to it and make necessary changes to the the coveralls builds running.

**Comment by greew** [comment]
Whoops.. sorry!! 
I meant to play around with it in a new branch - not push it to the pull request. Sorry about that :)

**Comment by greew** [comment]
And then...

The tests are passing and .travis.yml has been updated to reflect the newest testing as done in the core cakephp project.

Why the COVERALLS=1 test is failing, I've got no clue why.

**Comment by greew** [comment]
@AD7six Do you see any problems with this PR? If so, please let me know :)

File: .travis.yml [hunk 2]
```diff
@@ -19,12 +20,16 @@ matrix:
...
       env: PHPCS=1 DEFAULT=0
 
-    - php: 5.4
+    - php: 5.6
       env: COVERALLS=1 DEFAULT=0 DB=mysql db_dsn='mysql://travis@0.0.0.0/cakephp_test'
 
+  allow_failures:
+    - env: COVERALLS=1 DEFAULT=0 DB=mysql db_dsn='mysql://travis@0.0.0.0/cakephp_test'
+
+
```
Comment (ADmad) [inline .travis.yml]: Extra blank line

Comment (greew) [inline .travis.yml]: Done :)

**Pull request merged**